cogeo_mosaic/mosaic.py

```diff
@@ -271,7 +273,9 @@ def from_urls(
             >>> MosaicJSON.from_urls(["1.tif", "2.tif"])
 
         """
-        features = get_footprints(urls, max_threads=max_threads, quiet=quiet)
+        features = get_footprints(
+            urls, max_threads=max_threads, tms=tilematrixset, quiet=quiet
+        )
```

Comment:
tms will default to `WebMercatorQuad` directly in the `get_footprints` method

Reply:
not sure what you wanted changed here and in related comments. Do you want to add an inline comment?